lmfdb/hypergm/web_family.py

```diff
@@ -292,7 +292,7 @@ def default_prange(self):
 
     @lazy_attribute
     def maxp(self):
-        return -1 if not self.euler_factors.keys() else max(self.euler_factors.keys())
+        return -1 if not self.euler_factors else max(list(self.euler_factors))
```

Comment:
Same here: `max({1:2,3:4})` is `3`.  Though there are some cases where we might prefer to include `key` for clarity, since dictionaries are often used in contexts where you're thinking primarily of the values.